srcpkgs/superd/patches/ignore-binaries-on-install.diff

```diff
@@ -0,0 +1,18 @@
+Let xbps-src handle (cross)building super{d,ctl} binaries properly with the go build-style.
+
+--- a/Makefile
++++ b/Makefile
+@@ -58,12 +58,10 @@ doc: $(DOCS)
+ clean:
+ 	$(RM) $(DOCS) superd superctl
+ 
+-install: $(DOCS) superd superctl
++install: $(DOCS)
```

Comment:
Maybe you could try to upstream this by creating a new install-docs target and adding it to the install target's dependencies. The template could then just use `make install-docs` and we might be able to drop the patch eventually if it's accepted upstream.

Reply:
I came up with an alternative that lets up get rid of the patch entirely, let me know what you think :)